src/igvfd/types/file.py

```diff
@@ -1147,7 +1147,7 @@ def validate_bucket_location(request, properties, bucket):
 )
 def post_upload(context, request):
     properties = context.upgrade_properties()
-    if properties['upload_status'] == 'validated':
+    if properties['upload_status'] in ['validated', 'validation exempted']:
```

Comment:
I cannot imagine any other statuses, but wondering if explicitly spelling out the allowed ones (by saying `not in ['pending', 'invalidated', 'file not found']` which right now is of course equivalent but would not change in case of added statuses) would be more future proof?